bitbucket-pipelines.yml

```diff
@@ -13,16 +49,16 @@ pipelines:
             - npm install -g gulp-cli
             - gulp build
             - gulp publish --awsk $AWS_ACCESS_KEY_ID --awss $AWS_SECRET_ACCESS_KEY --production
-      # - step:   # Uncomment to add Cloudfront Invalidation step
-      #     name: Invalidate Cloudfront objects
-      #     script:
-      #       - pipe: atlassian/aws-cloudfront-invalidate:0.1.3
-      #         variables:
-      #           AWS_ACCESS_KEY_ID: $AWS_ACCESS_KEY_ID
-      #           AWS_SECRET_ACCESS_KEY: $AWS_SECRET_ACCESS_KEY
-      #           AWS_DEFAULT_REGION: $AWS_DEFAULT_REGION
-      #           DISTRIBUTION_ID: '$AWS_CF_DISTRIBUTION'
-      #           PATHS: '$AWS_CF_PATHS'
+      - step:
+          name: Invalidate Cloudfront objects
+          script:
+            - pipe: atlassian/aws-cloudfront-invalidate:0.4.1
+              variables:
+                AWS_ACCESS_KEY_ID: $AWS_ACCESS_KEY_ID
+                AWS_SECRET_ACCESS_KEY: $AWS_SECRET_ACCESS_KEY
+                AWS_DEFAULT_REGION: $AWS_DEFAULT_REGION
+                DISTRIBUTION_ID: '$AWS_CF_DISTRIBUTION'
+                PATHS: '$AWS_CF_PATHS'
```

Comment:
Can you please revert this? It's a configured part of the Mint pipeline, and not everyone will have CF invalidation setup.

Reply:
@kingo55 
Reverted.

Reply:
Thanks @allmywant - perfect...